common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -3213,6 +3214,14 @@ object CelebornConf extends Logging {
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefaultString("600s")
 
+  val CLIENT_FETCH_BUFFER_MAX_SIZE: ConfigEntry[Long] =
+    buildConf("celeborn.client.fetch.buffer.max.size")
+      .categories("client")
+      .version("0.4.0")
+      .doc("Max size of reducer partition buffer memory for shuffle reader.")
```

Comment:
Could you add more detail for this config like `celeborn.client.push.buffer.max.size` to tell user how to configure the reasonable config value?

Reply:
Updated. Do you have any suggestions for improving this config description?